must-gather/collection-scripts/gather_crds

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+source pwait
+max_parallelism=10
+
+# Cluster passed in from main gather
+clusterID=$1
+
+# Resource list
+resources=()
+
+# OADP
+for i in $(/usr/bin/oc get crd | grep oadp.openshift.io | awk '{print $1}'); do
```

Comment:
I know in other places there is mixture of `oc` and `/usr/bin/oc`, but please don't use full path as for example I have `oc` in venv and not in the /usr/bin/

Reply:
```
resources+=($(oc get crd | awk '/oadp.openshift.io/{print $1}'))
```

Reply:
is not this only run through container image? with `oc adm`?

Reply:
Possibly, but I would never assume that as certainty in scripts.

Reply:
did a grep and there are files that use `/usr/bin/oc` that I did not touch, so will add this to future task